.globalconfig

```diff
@@ -2,7 +2,7 @@ is_global = true
 
 # CA1000: Do not declare static members on generic types
```

Comment:
```suggestion
# CA1000: Do not declare static members on generic types
# https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1000
```

@iSazonov I think it would helpful to have documentation links to each rule in `.globalconfig`, do you agree?

Reply:
I think that would be a good addition

Reply:
Oh, in the case we should check the links in CI-Static.  There are frequent failures as it is. I don't think we need to make life difficult for us.

Reply:
I don't think it should be necessary to check the links in `CI-Static`. The URIs can be provided as a convenience rather than a normative reference.